.github/workflows/CI.yml

```diff
@@ -59,7 +59,7 @@ jobs:
       - name: Check doc aliases
         run: |
           python3 doc_aliases.py ../${{matrix.conf.name}}
-          cd .. && git diff --exit-code
+          cd .. && git config --global --add safe.directory `pwd` && git diff --exit-code
```

Comment:
I am not sure why this would be needed though

Reply:
The second checkout seems to overwrite `safe.directory`, I'll see if there is a more idiomatic way to stop this